homeassistant/components/mikrotik/__init__.py

```diff
@@ -49,6 +54,8 @@
             vol.Optional(CONF_ENCODING, default=DEFAULT_ENCODING): cv.string,
             vol.Optional(CONF_TRACK_DEVICES, default=True): cv.boolean,
             vol.Optional(CONF_ARP_PING, default=False): cv.boolean,
+            vol.Optional(CONF_SENSORS): vol.All(cv.ensure_list, [vol.In(SENSORS)]),
```

Comment:
All available sensor types should be created without an option.

See:
https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md

Reply:
Adding code to use all sensors by default